### PR TITLE
change video upload structure

### PR DESCRIPTION
**Pull request by gorkemgul**
**What's Changed?**

* Frame extraction, description extraction, and upload processes will be done one by one.

### CODE REVIEW

File: mapilio_kit/base/video_upload.py [hunk 1]
```diff
@@ -10,10 +12,16 @@ def fundamental_arguments(self, parser: argparse.ArgumentParser):
...
-        if not vars_args['processed']:
-            sampler().perform_task(vars_args)
-        else:
-            vars_args["import_path"]=vars_args["video_import_path"]
-        uploader().perform_task(vars_args)
+        vars_args_copy=deepcopy(vars_args)
+        for video_path in video_paths:
+            vars_args_copy["video_import_path"] = os.path.join(vars_args["video_import_path"],video_path)
+            if vars_args_copy["video_import_path"].endswith(video_formats):
+                if not vars_args['processed']:
```
Comment (bonj4) [inline mapilio_kit/base/video_upload.py:23]: in line  23 vars_args has to be replaced with vars_args_copy

Comment (gorkemgul) [reply mapilio_kit/base/video_upload.py]: nice catch, thanks :tada:

**Review by bonj4 — APPROVED** [review]

File: mapilio_kit/base/video_upload.py [hunk 2]
```diff
@@ -10,10 +12,16 @@ def fundamental_arguments(self, parser: argparse.ArgumentParser):
...
+        from . import uploader, sampler
+        video_paths = os.listdir(vars_args["video_import_path"]) if os.path.isdir(vars_args["video_import_path"]) else [vars_args["video_import_path"]]
+        video_formats = ('.mp4', '.avi', '.mov', '.mkv', '.360', '.MP4', '.AVI', '.MOV', '.MKV')
 
-        if not vars_args['processed']:
-            sampler().perform_task(vars_args)
-        else:
-            vars_args["import_path"]=vars_args["video_import_path"]
-        uploader().perform_task(vars_args)
+        vars_args_copy=deepcopy(vars_args)
```
Comment (ozcan-durak) [inline mapilio_kit/base/video_upload.py:19]: Why using deepcopy ?

Comment (gorkemgul) [reply mapilio_kit/base/video_upload.py:19]: Because some arguments needs to be overwritten

**Pull request merged**